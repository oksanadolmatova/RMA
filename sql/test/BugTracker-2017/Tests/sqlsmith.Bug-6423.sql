select  
  cast(nullif(sample_4.id,
    ref_5.name) as clob) as c0, 
  ref_7.type as c1, 
  sample_4.ds as c2, 
  65 as c3
from 
  tmp.objects as sample_0
    inner join sys.environment as ref_5
        left join sys.tables as ref_6
        on ((76 is not NULL) 
            or ((true) 
              and ((ref_6.name is NULL) 
                or (ref_6.access is NULL))))
      inner join sys.storagemodel as ref_7
        left join bam.rg as sample_4
          right join sys.key_types as ref_8
          on (true)
        on (true)
      on (ref_6.commit_action = ref_8.key_type_id )
    on (sample_4.lb is not NULL)
where ref_7.columnsize is not NULL
limit 85;
