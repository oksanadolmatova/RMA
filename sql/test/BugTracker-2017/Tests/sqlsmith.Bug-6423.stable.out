stdout of test 'sqlsmith.Bug-6423` in directory 'sql/test/BugTracker-2017` itself:


# 13:47:15 >  
# 13:47:15 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=35547" "--set" "mapi_usock=/var/tmp/mtest-16781/.s.monetdb.35547" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 13:47:15 >  

# MonetDB 5 server v11.27.6
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2017', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 7.330 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:35547/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-16781/.s.monetdb.35547
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 13:47:16 >  
# 13:47:16 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-16781" "--port=35547"
# 13:47:16 >  

#select  
#  cast(nullif(sample_4.id,
#    ref_5.name) as clob) as c0, 
#  ref_7.type as c1, 
#  sample_4.ds as c2, 
#  65 as c3
#from 
#  tmp.objects as sample_0
#    inner join sys.environment as ref_5
#        left join sys.tables as ref_6
#        on ((76 is not NULL) 
#            or ((true) 
#              and ((ref_6.name is NULL) 
#                or (ref_6.access is NULL))))
#      inner join sys.storagemodel as ref_7
#        left join bam.rg as sample_4
#          right join sys.key_types as ref_8
#          on (true)
% .L53,	.L55,	.L57,	.L61 # table_name
% c0,	c1,	c2,	c3 # name
% clob,	clob,	clob,	tinyint # type
% 0,	0,	0,	1 # length

# 13:47:16 >  
# 13:47:16 >  "Done."
# 13:47:16 >  

